lifecycle-operator/controllers/lifecycle/keptntask/job_utils_test.go

```diff
@@ -208,8 +208,11 @@ func TestKeptnTaskReconciler_generateJob(t *testing.T) {
 	taskDefinitionName := "my-task-definition"
 	token := true
 	var ttlSecondsAfterFinished int32 = 100
+	imagePullSecret := []v1.LocalObjectReference{{
+		Name: "my-docker-secret",
+	}}
 
-	taskDefinition := makeTaskDefinitionWithServiceAccount(taskDefinitionName, namespace, svcAccname, &token, &ttlSecondsAfterFinished)
+	taskDefinition := makeTaskDefinitionWithServiceAccount(taskDefinitionName, namespace, svcAccname, &token, &ttlSecondsAfterFinished, imagePullSecret)
```

Comment:
please, add later at around line 250 an assertion that checks for the secrets to be assigned correctly